server/routes/milestone-report.js

```diff
@@ -8,21 +8,24 @@ const logger = require('../logger.js');
 const { asyncMiddleware } = require('../error-handler.js');
 
 // Service
-const { getReport, getHiredParticipantsReport } = require('../services/reporting');
+const {
+  getReport,
+  getHiredParticipantsReport,
+  getHiredHealthRegionParticipantsReport,
+} = require('../services/reporting');
 
 // Router
 const router = express.Router();
 
-router.use(keycloak.allowRolesMiddleware('ministry_of_health'));
-router.use(keycloak.getUserInfoMiddleware());
-
 router.get(
   '/',
+  [keycloak.allowRolesMiddleware('ministry_of_health'), keycloak.getUserInfoMiddleware()],
   asyncMiddleware(async (req, res) => res.status(200).json({ data: await getReport() }))
 );
 
 router.get(
   '/csv/hired',
```

Comment:
We can reuse this route and service method.

Reply:
I think we need a new endpoint

Reply:
Actually, I'm curious - what would work best in this scenario?
- using the same endpoint for both HA and MoH (_assuming that validation done on FE_); or
- switch to different endpoints for different roles ?

Reply:
if the underlying service method and processing is context-driven, then we can use the same route and service method.